### DISCUSSION TITLE
Issue: Unable to Save Settings During Initial Setup

### DISCUSSION THREAD

**Discussion by discourse-monger** [nextai-translator/nextai-translator#1734] [Q&A]
During the initial setup of the application, after clicking the "Save" button, nothing happens. The loading spinner keeps spinning indefinitely. Could you please help me resolve this issue? Thank you in advance!
![Screenshot_1](https://github.com/user-attachments/assets/57a90fdb-9e5b-4eb7-8786-ae007d00a81e)

**Comment by Foryings** [comment]
I also encountered this problem, downloading the old version doesn't work either

**Reply by myd7349** [reply]
https://github.com/openai-translator/openai-translator/issues/1735

**Comment by ws153759** [comment]
I also encountered the same issue on my new desktop. Haven't fixed it yet.
